packages/react/src/drafts/TabPanels/TabPanels.docs.json

```diff
@@ -0,0 +1,72 @@
+{
+  "id": "tab_panels",
+  "name": "TabPanels",
+  "status": "draft",
+  "a11yReviewed": false,
+  "stories": [],
+  "props": [
+    {
+      "name": "aria-label",
+      "type": "string",
+      "description": "Used to set the `aria-label` on the `role=\"tablist\"` element."
+    },
+    {
+      "name": "children",
+      "type": "React.ReactNode",
+      "description": "The content of the component, can contain Tabs, Panels but also content before and after Tabs and after the Panels."
+    },
+    {
+      "name": "id",
+      "type": "string",
+      "description": "The id of the tab container, used to generate child ids."
+    },
+    {
+      "name": "selectedTabIndex",
+      "type": "number",
+      "description": "The 0-based index of the tab that is selected."
+    },
+    {
+      "name": "onTabContainerChange",
+      "type": "function",
+      "description": "Callback fired when the tab container changes (bubbles, cancelable): fired on `<tab-container>` before a new tab is selected and visibility is updated. `event.tab` is the tab that will be focused and `tab.panel` is the panel that will be shown if the event isn't cancelled."
+    },
+    {
+      "name": "onTabContainerChanged",
+      "type": "function",
+      "description": "Callback fired when the tab container changes (bubbles): fired on `<tab-container>` after a new tab is selected and visibility is updated. `event.tab` is the tab that is now active (and will be focused right after this event) and `event.panel` is the newly visible tab panel."
+    },
```

Comment:
I don't see where these handlers get bound - does that happen automatically with `createComponent`?

Reply:
yes. Specifically [we made them props on the component](https://github.com/github/tab-container-element/blob/984ef06309b29797ed52965f08135fcb94b52a8e/src/tab-container-element.ts#L52-L68).

Reply:
[I've added an alias called `onChange` and `onChanged`](https://github.com/github/tab-container-element/blob/main/src/tab-container-element.ts#L34) so we're documenting that now instead (same binding, it's a prop on the underlying web component).